jsk_perception/node_scripts/alexnet_object_recognition.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from chainer import cuda
+import chainer.serializers as S
+from chainer import Variable
+import numpy as np
+import skimage.transform
+
+import cv_bridge
+from jsk_recognition_msgs.msg import ClassificationResult
+from jsk_recognition_utils.chainermodels import Alex
+from jsk_recognition_utils.chainermodels import AlexBatchNormalization
+from jsk_topic_tools import ConnectionBasedTransport
+from jsk_topic_tools.log_utils import logerr_throttle
+import message_filters
+import rospy
+from sensor_msgs.msg import Image
+
+
+class AlexNetObjectRecognition(ConnectionBasedTransport):
```

Comment:
Maybe you can inherit `VGG16ObjectRecognition` in `vgg16_object_recognition.py`?

```python
from vgg16_object_recognition import VGG16ObjectRecognition

class AlexNetObjectRecognition(VGG16ObjectRecognition):
    def __init__(self):
        super(VGG16ObjectRecognition, self).__init__()  # initialize ConnectionBasedTransport only
        ...   # initialize AlexNetObjectRecognition
```